airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -488,11 +488,14 @@ def test_check(self, connector_config, inputs: ConnectionTestConfig, docker_runn
             assert len(con_messages) == 1, "Connection status message should be emitted exactly once"
             assert con_messages[0].connectionStatus.status == Status.FAILED
         elif inputs.status == ConnectionTestConfig.Status.Exception:
-            with pytest.raises(ContainerError) as err:
-                docker_runner.call_check(config=connector_config)
-
-            assert err.value.exit_status != 0, "Connector should exit with error code"
-            assert "Traceback" in err.value.stderr, "Connector should print exception"
+            output = docker_runner.call_check(config=connector_config, raise_container_error=False)
+            trace_messages = filter_output(output, Type.TRACE)
+            assert len(trace_messages) == 1, "A trace message should be emitted in case of unexpected errors"
+            trace = trace_messages[0].trace
+            assert isinstance(trace, AirbyteTraceMessage)
+            assert trace.error is not None
+            assert trace.error.message is not None
+            assert trace.error.internal_message is not None
```

Comment:
Do we want to enforce `internal_message` even for connectors not using CDK tooling to create their traces? At the moment, the airbyte protocol only enforces `message` as a required property.

Reply:
Good call! I removed the assertion on `internal_message`